.gitignore

```diff
@@ -11,3 +11,7 @@ obj/
 /Build/*.debug.cfg
 
 *.user
+/Build/TestSound
+/Build/Bloodmasters.cfg
```

Comment:
Please do not add the `.cfg` file here and don't remove it from the project. We need the config file.